src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -741,9 +722,9 @@ BuildAndGetResponseFactorsObjectFromSingleBHs(EnergyPlusData &state, std::vector
 
 //******************************************************************************
 
-void SetupBHPointsForResponseFactorsObject(std::shared_ptr<GLHEResponseFactors> &thisRF)
+void SetupBHPointsForResponseFactorsObject(const std::shared_ptr<GLHEResponseFactors> &thisRF)
 {
-    for (auto &thisBH : thisRF->myBorholes) {
+    for (auto const &thisBH : thisRF->myBorholes) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:746]:(style),[constVariable],Variable 'thisBH' can be declared as reference to const